bpf/aya-bpf/src/args.rs

```diff
@@ -130,6 +150,15 @@ impl<T> FromPtRegs for *const T {
         }
     }
 
+    fn from_stack_argument(ctx: &pt_regs, n: usize) -> Option<Self> {
+        unsafe {
+            let addr: c_ulonglong = &ctx.uregs[13] + 8 * (n + 1) as c_ulonglong;
```

Comment:
how come this one is a reference and all the others are not? looks like aarch64 may be implemented incorrectly.